packages/errors/__snapshot-html__/EXPERIMENTAL_SKIP_DOMAIN_INJECTION.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+    <html>
+    <head>
+      <meta charset="utf-8" />
+      
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Courier+Prime&display=swap" rel="stylesheet">
+    <style>
+  
+        body {
+          font-family: "Courier Prime", Courier, monospace;
+          padding: 0 1em;
+          line-height: 1.4;
+          color: #eee;
+          background-color: #111;
+        }
+        pre {
+          padding: 0 0;
+          margin: 0 0;
+          font-family: "Courier Prime", Courier, monospace;
+        }
+      
+    body {
+      margin: 5px;
+      padding: 0;
+      overflow: hidden;
+    }
+    pre {
+      white-space: pre-wrap;
+      word-break: break-word;
+      -webkit-font-smoothing: antialiased;
+    }
+    </style>
+  
+    </head>
+    <body><pre><span style="color:#e05561">The <span style="color:#e5e510">experimentalSkipDomainInjection<span style="color:#e05561"> experiment is over. <span style="color:#e5e510">document.domain<span style="color:#e05561"> injection is now off by default.<span style="color:#e6e6e6">
```

Comment:
should we point users to the `injectionDocumentDomain` option?

Reply:
I don't think so - this experiment basically partially applied the new default behavior to a selection of hostnames. There would be no reason to set `injectDocumentDomain` to true, unless they were _also_ relying on `cy.origin()` being unnecessary for subdomains on a different set of hostnames - in which case, they'll get an error message from the cy.origin command.